api/tools/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+debug=true
```

Comment:
Is there a way to hook up the intelliJ debugger? I've been making do this print statements... being able to use the debugger would be awesome.

Reply:
I haven't personally done this or tried to do it, may be possible though. This line really just tells Spring to generate a lot of log spam (including dependency injection details)